peer/store-utils.go

```diff
@@ -11,12 +11,21 @@ import (
 	"github.com/gluster/glusterd2/store"
 
 	log "github.com/Sirupsen/logrus"
+	"github.com/pborman/uuid"
 )
 
 const (
 	peerPrefix string = store.GlusterPrefix + "peers/"
 )
 
+var (
```

Comment:
The comments were actually in the commit that introduced this. https://github.com/gluster/glusterd2/pull/90/commits/74190809a3c1525ca5164ddeb8940d4d9a86aa88

This was  done to allow these functions to be mocked during unit testing.